### DISCUSSION TITLE
About new beginners project

### DISCUSSION THREAD

**Discussion by Aniruddha120** [lava-nc/lava#526] [Q&A]
Dear all,
I don't know whether this topic fits here or not, but I wanted to ask anyways. I just went through the documentation and I am a beginner in neuromorphic domain. Can you please suggest me idea of beginner level project ideas? I know asking for ideas is also sometimes unethical, but as I am a beginner, I really don't have that skill to deep dive but I want to get the taste of completing a project. Hope you can understand. Thanks!

**Comment by awintel** [comment]
Thank you for your interest in contributing to Lava. We certainly welcome contributors like you.
What is your area of interest and expertise?

As a beginner, it might make the most sense to develop use-cases or new tutorials to help make Lava more accessible to others as well. 
- You could train a deep event-based neural network for some video or audio processing task with lava-dl. 
- Or you could develop an online learning algorithm from the literature. 
- Or you could apply the Lava QUBO solver to a public benchmark. 

As you gain user experience while solving a real problem, you'll gain competence to contribute at a lower SW level as well if you are interested.
